### ISSUE TITLE
ID on Canvas in main HTML

### ISSUE THREAD

**Issue by Eliora-Horst** [Eliora-Horst/CS424Project#4]
In order to run the game system, the ID on the canvas needs to be unique to JavaScript engine running, so any styling on the canvas element should be done through a class or a higher level ID.

**Comment by noorka** [comment]
Styling including the size/ placement of canvas?

**Comment by Eliora-Horst** [comment]
Yes, though for now let's leave sizing of the canvas on the <canvas> element itself, as that is how it's being done in the book.  For the book project, the size of the canvas is 640x480, if you need that reference for centering it 😃

**Comment by noorka** [comment]
Ok. I will spiff up the canvas for the next commit of the HTML
